backend-service/src/common/interfaces/index.interface.ts

```diff
@@ -0,0 +1,13 @@
+export interface Token {
+  name: string;
+  symbol: string;
+  decimals: number;
+  description?: string;
+  website?: string;
+  twitter?: string;
+  tokens: {
+    [key: string]: {
```

Comment:
can we have a custom type for key? Like the should be any of this
<img width="931" alt="Screen Shot 2024-03-19 at 4 09 21 PM" src="https://github.com/njokuScript/base-migrate/assets/38771353/17cbb3ce-d7b5-47bb-b1d1-5476204e1ce9">

Reply:
> can we have a custom type for key? Like the should be any of this <img alt="Screen Shot 2024-03-19 at 4 09 21 PM" width="931" src="https://private-user-images.githubusercontent.com/38771353/314248506-17cbb3ce-d7b5-47bb-b1d1-5476204e1ce9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA5MjYwNjcsIm5iZiI6MTcxMDkyNTc2NywicGF0aCI6Ii8zODc3MTM1My8zMTQyNDg1MDYtMTdjYmIzY2UtZDdiNS00N2JiLWIxZDEtNTQ3NjIwNGUxY2U5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIwVDA5MDkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU1ZTJjZjk1OTJiZTFlY2M1OWRlNjllOTUwZjlhNTNiMDJkOTk4MDk4MGM3Zjc0MTZmNmY0ODQyZjQzOGE2Y2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3prjpTzUsoqYtjPTV1IaC6dmtPX4D2W3vJVVVTSNPq0">

Yeah, i was concern about not enforcing it but i will just make them optional